website/static/js/cards.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+     window.cancelProp = function(e) {
+        e.cancelBubble = true;
+        if (e.stopPropagation) {
+            e.stopPropagation();
+        }
+    };
+
+    window.toggleIcon = function(el) {
+        jQuery(el.querySelector('.toggle-icon')).toggleClass('fa-angle-down fa-angle-up');
+    };
+});
```

Comment:
This file  is not a good place to have these functions. They are global which is a problem but also this is not very reusable. We have utility js files to house these kinds of things, especially for cancelling propagation. The toggleIcon function is too specific and needs to be in context where the toggling of view also is happening. I don't imagine people would need a stand alone icon toggle function.